lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -2078,12 +2090,55 @@ void GrandCentralPass::runOnOperation() {
     VerbatimBuilder::Base verbatimData;
     VerbatimBuilder verbatim(verbatimData);
     verbatim += instanceSymbol;
-    auto iface =
-        traverseBundle(bundle, bundle.getID(), bundle.getPrefix(), verbatim);
+    // List of interface elements.
+    SmallVector<TypeSum> interfaceElems;
+    // insertedOps will record all the verbatim ops inserted in the companion
+    // module, which must be erased, if the interface is redundant. The
+    // traverseBundle is a recursive traversal of the AugmentedBundleType, it
+    // will construct the Interface and populate it with the elements and insert
+    // the corresponding verbatim xmr assignments. The interface might be
+    // redundant if it is a nonlocal view and another exactly same interface is
+    // already generated. To compare the interface elements, with an existing
+    // interface, it is necessary to complete the traverseBundle traversal and
+    // gather all the elements of the interface.
```

Comment:
Can the PR include a `TODO` that this is something that we want to change in the future?

Reply:
Added the TODO.